src/test/formatter.cc

```diff
@@ -122,10 +133,10 @@ TEST(XmlFormatter, DumpStream3) {
 
   fmt.open_array_section("foo");
   fmt.dump_stream("blah") << "hithere";
-  fmt.dump_float("pi", 3.14);
```

Comment:
unfortunatelly, 3.14 can not be represented as `double`. It will look like `3.1400000000000001`